python_modules/dagster-test/dagster_test/toys/auto_materializing/assets.py

```diff
@@ -0,0 +1,86 @@
+from dagster import AssetKey, AutoMaterializePolicy, SourceAsset, asset, repository
+from dagster._core.definitions.freshness_policy import FreshnessPolicy
+
+# EAGER
+
+
+@asset(auto_materialize_policy=AutoMaterializePolicy.lazy())
+def eager_upstream():
```

Comment:
I find it a little confusing that this asset with `eager` in the name has a lazy materialization policy.